internal/pkg/db/redis/scripts.go

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright 2018 Redis Labs Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package redis
+
+import "github.com/gomodule/redigo/redis"
+
+// **********************
+// Lua scripts notes:
+// * magic number 4096 is less < 8000 (redis:/deps/lua/lapi.c:LUAI_MAXCSTACK -> unpack error)
+// * assumes a single instance
+// * `get*`` scripts are implementations for range operations. To be used when the server is remote
+
+const (
+	scriptGetObjectsByRange       = "getObjectsByRange"
+	scriptGetObjectsByRangeFilter = "getObjectsByRangeFilter"
+	scriptGetObjectsByScore       = "getObjectsByScore"
+	scriptUnlinkZsetMembers       = "unlinkZsetMembers"
+	scriptUnlinkCollection        = "unlinkCollection"
+)
+
+var scripts = map[string]redis.Script{
+	scriptGetObjectsByRange: *redis.NewScript(1, `
+		local magic = 4096
```

Comment:
I actually ment moving whole scripts as string constants to the top the file. I am not sure, but I think this will improve redability and editiability. I leave this to your choice.

Reply:
Ok, I see what you meant :) Fixed.